src/components/chat/ChatCardList.tsx

```diff
@@ -0,0 +1,57 @@
+//style
+import styled from "styled-components";
+//component
+import ChatCard from "./ChatCard";
+import { useRecoilValue } from "recoil";
+import {
+  chatRoomListState,
+  nameSearchState,
+  usersState,
+} from "../../recoil/state";
+// 나중에 index 말고 value로 넘겨서 order에 맞게 최신순대로 출력, order 배열 따로 만들어서 그에 맞는
+//chatcard 출력
+const ChatCardList = () => {
+  const chatList = useRecoilValue(chatRoomListState);
+  const searchInput = useRecoilValue(nameSearchState);
+  const users = useRecoilValue(usersState);
+  const input = useRecoilValue(nameSearchState);
+  //검색 이름에 해당하는 card 만 true return
+  const nameSearch = (userId: number) => {
+    if (users[userId].name.includes(input)) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
```suggestion
   return users[userId].name.includes(input);
```
이렇게 줄여서 사용할 수 있을 것 같아요!